roles/api-environment/templates/express-api/start-express-container.sh.j2

```diff
@@ -40,36 +25,30 @@ if [[ "$(docker ps -aq -f name=$api_name)" ]]; then
     done
 fi
 
-# Remove image if it already exists
-image_id=$(docker images -q $api_name)
-if [[ $image_id ]]; then
-  echo "Remove existing image $image_id"
-  docker rmi $image_id --force
-fi
-
 # Create logs folder and make sure the API have access to wrtie logs into it
 mkdir -p $api_path/logs
 chown -R {{ api_user }}:{{ api_group }} $api_path/logs
 
-# Load docker image from the artifact
-docker load --input $archive
-
 # Run API in a container
 uid="$(id -u {{ api_user }})"
 gid="$(id -g {{ api_user }})"
 docker run -d \
            --user "$uid:$gid" \
            --workdir /usr/src/$api_name \
+           --env API_NAME=$api_name \
+           --env FOREVER_ROOT=/usr/src/$api_name/.forever \
+           --env HOME=/usr/src/$api_name \
```

Comment:
Why does `HOME` need to be set here?

Reply:
If this is not set, yarn attempts to write to the container's read only file system. Setting home to the writable volume prevents a series of repeated warning from yarn about this.

Reply:
Using YARN_CACHE_FOLDER works for the run docker image as installation has already occurred. In the build docker image, we still need to set HOME to set the global to set the global install directory to a writable volume, which [is not easily configurable with other methods](https://github.com/yarnpkg/yarn/issues/5746)